test/e2e/storage/testsuites/provisioning.go

```diff
@@ -327,6 +327,26 @@ func (p *provisioningTestSuite) DefineTests(driver storageframework.TestDriver,
 		}
 		wg.Wait()
 	})
+
+	ginkgo.It("should mount multiple PV pointing to the same storage on the same node", func() {
+		// csi-hostpath driver does not support this test case. In this test case, we have 2 PV containing the same underlying storage.
+		// during the NodeStage call for the second volume, csi-hostpath fails the call, because it thinks the volume is already staged at a different path.
+		// Note: This is not an issue with driver like PD CSI where the NodeStage is a no-op for block mode.
+		if pattern.VolMode == v1.PersistentVolumeBlock {
+			e2eskipper.Skipf("skipping multiple PV mount test for block mode")
```

Comment:
Skipping *why"? The message doesn't add any information that itsn't already available ("skipped" is printed by e2eskipper.Skipf and the rest is part of the test name).